api/definition/swagger.yaml

```diff
@@ -0,0 +1,273 @@
+        definition:
+          type: string
+          example: "The percentage of patients with established hypertension, as
+              recorded on practice disease registers (proportion of total list
+              size)"
+          description: The definition of the indicator
+    HealthDataPoint:
+      type: object
+      description: Represents a health data point for a public health indicator with a
+        count, value, upper confidence interval, lower confidence interval and a
+        year.
+      required:
+        - year
+        - count
+        - value
+        - lowerCi
+        - upperCi
+      properties:
+        year:
+          type: integer
+          format: int32
+          example: 2023
+          description: The year that the data point is for
+        count:
+          type: number
+          format: int32
+          example: 222
+          description: The count
+        value:
+          type: number
+          format: int32
+          example: 506.60912
+          description: The value
+        lowerCi:
+          type: number
+          format: int32
+          example: 441.69151
+          description: The lower confidence interval
+        upperCi:
+          type: number
+          format: int32
+          example: 578.32766
+          description: The upper confidence interval
+    HealthDataForIndicatorAndArea:
+      type: object
+      description: Associates a list of health data points with the relevant
+        geographical area (represented by it's unique code) and indicator
+        (represented by it's unique identifier).
+      required:
+        - areaCode
+        - healthData
+        - indicatorId
+      properties:
+        areaCode:
+          type: string
+          example: A1426
+          description: The unique area code that the health data is are for
+        indicatorId:
+          type: integer
+          format: int32
+          example: 1234
+          description: The unique identifier of the indicator that the health data points are for
+        healthData:
+          type: array
+          items:
+            $ref: "#/components/schemas/HealthDataPoint"
+          description: The health data points for the area and indicator
+  parameters:
+    years:
+      in: query
+      name: years
+      description: A list of years, up to 10 years can be requested
+      style: form
+      schema:
+        type: array
+        items:
+          type: integer
+          example: 2023
+        maxItems: 10
+    area_codes:
+      in: query
+      name: area_codes
+      description: A list of area codes, up to 10 area codes can be requested
+      style: form
+      schema:
+        type: array
+        items:
+          type: string
+          example: G82109
+        maxItems: 10
+    indicator_id:
+      in: path
+      name: indicator_id
+      style: simple
+      schema:
+        type: integer
+        example: 1234
+        minimum: 1
+      explode: false
+      required: true
```

Comment:
In the documentation generated by VS Code at least, this `required` parameter doesn't appear to be represented in the same was as for path parameters, and it allows you to use the "try it" button to call the API without providing a value. Is it valid to specify a query parameter as required?